src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -538,7 +603,7 @@ class SnykSettingsDialog(
         rootPanel.add(
             userExperiencePanel,
             baseGridConstraints(
-                row = 5,
+                row = 7,
```

Comment:
What do these layout changes do? We didn't add anything to the root level, right?

Reply:
Fixed!